ci/aws-analytical-env/jobs/terminate-cluster/stop_prod.yml

```diff
@@ -8,6 +8,15 @@ jobs:
           params:
             AWS_ROLE_ARN: arn:aws:iam::((aws_account.production)):role/ci
 
+  - name: prod-stop-user-batch-cluster
+    plan:
+      - get: aws-analytical-env
+        trigger: false
+      - .: (( inject meta.plan.stop-user-cluster ))
+        config:
+          params:
+            AWS_ROLE_ARN: arn:aws:iam::((aws_account.production)):role/ci
+
```

Comment:
Should we not be putting this in preprod too? Genuine question.

Reply:
Its only an issue in production where users are runnning multiday jobs. In other environments, either we dont run jobs, or they run for about 10 mins, then shutdown due to inactivity.